db/sql/05_msar.sql

```diff
@@ -3419,7 +3419,7 @@ BEGIN
     END LOOP;
   END IF;
   column_defs := __msar.process_col_def_jsonb(0, col_defs, false);
-  PERFORM msar.add_temp_table(tab_name, column_defs);
+  PERFORM msar.add_temp_table(uq_tab_name, column_defs);
```

Comment:
In the process of fixing these tests I was actually able to find a find a bug in the case when a `tab_name` isn't provided to `prepare_temp_table_for_import`, i.e. `tab_name` is `null`, this is fixed by passing `uq_tab_name` for `msar.add_temp_table`.